test/unit/test_translate.py

```diff
@@ -27,16 +25,6 @@
             "Test file line 2\n"
 
 
-@pytest.fixture
```

Comment:
why do we want to remove these?

Reply:
Why would it matter? The test pass as is and there are no fixtures needed. Imo, these just affected readability negatively and didn't really have a useful purpose.

Reply:
just wondering. That's true, given the one liners, they did not add a ton of value.